nengo_loihi/config.py

```diff
@@ -189,3 +189,6 @@ def set_defaults():
     """
     nengo.Ensemble.max_rates.default = nengo.dists.Uniform(100, 120)
     nengo.Ensemble.intercepts.default = nengo.dists.Uniform(-1.0, 0.5)
+    nengo.Ensemble.neuron_type.default = nengo.LIF(
```

Comment:
This is just setting the default neuron type for Ensembles.

I guess we could do something like `nengo.SpikingRectifiedLinear.state["voltage"] = Choice([0])`, and the same for `LIF`. I hadn't really thought of that.

What I've been doing (as you probably noticed) is to prefer `LoihiLIF` and `LoihiSpikingRectifiedLinear` to their `nengo` counterparts; when you're specifying neurons on Loihi, they're treated the same, but the Loihi ones have initial voltages all zero. We might not want to encourage users to do that through the config system, though, because then it might apply to neurons that are off-chip, too (causing them to behave like Loihi neurons unnecessarily).

Reply:
Right yeah your suggestion addresses what I wanted, rather than what I said 😄